test/NuGet.Core.Tests/NuGet.Protocol.VisualStudio.Tests/project.json

```diff
@@ -1,6 +1,9 @@
 ﻿{
   "version": "1.0.0-*",
+  "compile": "*.cs",
```

Comment:
If you use the directory structure compiler/resources/ you can copy the resources there and it will not needed any changes to the project.json file. See NuGet.Commands.Test